src/ceph-volume/ceph_volume/tests/functional/batch/playbooks/test.yml

```diff
@@ -23,6 +26,12 @@
   become: yes
   tasks:
 
+    - name: rescan pvs
+      command: "pvscan --cache"
+    - name: rescan vgs
+      command: "vgscan"
+    - name: list lvs
+      command: "lvs"
```

Comment:
I'm just curious about this task ? What's the purpose of this ? only printing information ?
Or should this be modified by `lvscan` ?

Reply:
As per IRC conversation:

> [09:24:38] <jafaj> guits: morning. since we create "fake nvmes" on loop devices, lvm sees a given PV on two devices. Changing lvm.conf should avoid this https://github.com/ceph/ceph/pull/33391/files#diff-6ea0c4a596b89065b807a3108550c0faR10-R21 but I have found that to be not very reliable. Rescaning can help and listing lvs is more of a logging output
[09:25:10] <jafaj> to immediately see if lvm got confused or if a failure is actually a failure

Reply:
:+1: